vendor/github.com/containers/common/pkg/parse/parse.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // ValidateVolumeOpts validates a volume's options
 func ValidateVolumeOpts(options []string) ([]string, error) {
-	var foundRootPropagation, foundRWRO, foundLabelChange, bindType, foundExec, foundDev, foundSuid, foundChown, foundUpperDir, foundWorkDir, foundCopy, foundCopySymlink int
+	var foundRootPropagation, foundRWRO, foundLabelChange, bindType, foundExec, foundDev, foundSuid, foundChown, foundUpperDir, foundWorkDir, foundCopy, foundCopySymlink, foundVolumeSubpath int
```

Comment:
these changes must go in the https://github.com/containers/common project and then revendor them

Reply:
I understand that, but I need some details.

Is the correct procedure here as follows?

* Move this commit into separate PR in containers/common
* Wait for it to be merged
* `go get -u github.com/containers/common && go mod vendor` in this PR
* Rebase this PR with new `go.{mod,sum}` and `vendor`

Reply:
@giuseppe bump?

Reply:
sorry, forgot to submit the review.